test/route-rate-limit.test.js

```diff
@@ -780,6 +780,39 @@ test('global timeWindow when not set in routes', t => {
   })
 })
 
+test('timeWindow specified as a string', t => {
```

Comment:
The test is passing also without the change, we need to use a custom store in order to test this issue.

Reply:
Alright yep you are right, fails when running standalone, but as a test passes regardless, will change to use a custom store and look at `x-ratelimit-reset` to verify the timeWindow is has been converted correctly.